frontend/src/scenes/pipeline/Overview.tsx

```diff
@@ -0,0 +1,273 @@
+    </LemonCard>
+)
+
+const PipelineStepTransformation = ({
+    transformation,
+}: {
+    transformation: PluginConfigWithPluginInfoNew
+}): JSX.Element => {
+    let metrics: StatusMetrics | undefined = undefined
+
+    const { appMetricsResponse } = useValues(pipelineAppMetricsLogic({ pluginConfigId: transformation.id }))
+    const { sortedEnabledPluginConfigs } = useValues(pipelineTransformationsLogic)
+
+    if (appMetricsResponse) {
+        metrics = {
+            totals: appMetricsResponse.metrics.totals.successes + appMetricsResponse.metrics.totals.failures,
+            failures: appMetricsResponse.metrics.totals.failures,
+        }
+    }
+
+    return (
+        <PipelineStep
+            order={
+                transformation.enabled
+                    ? sortedEnabledPluginConfigs.findIndex((pc) => pc.id === transformation.id) + 1
+                    : undefined
+            }
+            name={transformation.name}
+            to={urls.pipelineApp(PipelineAppKind.Transformation, transformation.id, PipelineAppTab.Configuration)}
+            description={transformation.description}
+            enabled={transformation.enabled}
+            metrics={metrics}
+            headerInfo={
+                <>
+                    <div className="mr-1">
+                        <AppMetricSparkLine config={transformation} />
+                    </div>
+                    <More overlay={<TransformationsMoreOverlay pluginConfig={transformation} />} />
+                </>
+            }
+        />
+    )
+}
+
+const PipelineStepDestination = ({ destination }: { destination: DestinationType }): JSX.Element => {
+    let metrics: StatusMetrics | undefined = undefined
+    if (destination.backend !== PipelineAppBackend.BatchExport) {
+        const logic = pipelineAppMetricsLogic({ pluginConfigId: destination.id })
+        const { appMetricsResponse } = useValues(logic)
+
+        if (appMetricsResponse) {
+            metrics = {
+                totals: appMetricsResponse.metrics.totals.successes + appMetricsResponse.metrics.totals.failures,
+                failures: appMetricsResponse.metrics.totals.failures,
+            }
+        }
+    }
+
+    return (
+        <PipelineStep
+            name={destination.name}
+            to={destination.config_url}
+            description={destination.description}
+            enabled={destination.enabled}
+            metrics={metrics}
+            headerInfo={
+                <>
+                    <div className="mr-1">
+                        <AppMetricSparkLine config={destination} />
+                    </div>
+                    <More overlay={<DestinationMoreOverlay destination={destination} />} />
+                </>
+            }
+            additionalInfo={
+                <div className="flex gap-1">
+                    {destination.backend === PipelineAppBackend.Plugin ? (
+                        <LemonTag type="primary">Plugin</LemonTag>
```

Comment:
let's remove this - it's not relevant to the user, if we want to keep something I'd do frequency "realtime" vs "hourly" vs "daily" vs ...